htdocs/product/stock/product.php

```diff
@@ -1132,6 +1132,7 @@
 				}
 				foreach ($details as $pdluo) {
 					$product_lot_static->id = $pdluo->lotid;
+					$product_lot_static->fk_product = $pdluo->fk_product_stock;
```

Comment:
fk_product_stock is a rowid in into llx_product_stock
but fk_product is an id of product . Are you sure ?

Reply:
i remove and will check later